lib/home/profile/account/account_settings.dart

```diff
@@ -93,8 +95,11 @@ class _AccountSettingsState extends State<AccountSettings> {
               ),
             _ => ListTile(
                 leading: const Icon(Icons.person_off_outlined),
-                title: const Text('close account'),
-                onTap: () => navigator.showDialog(const CloseAccount()),
+                title: const Text('Close Account'),
+                onTap: () => showDialog(
+                  context: context,
+                  builder: (context) => const DeleteAccountDialog(), 
+                ),
```

Comment:
Oh dang, looks like with this change, we now have [**delete_account.dart**](https://github.com/theheartcenter-inc/thc/blob/manage-users/lib/home/profile/account/delete_account.dart) and [**close_account.dart**](https://github.com/theheartcenter-inc/thc/blob/manage-users/lib/home/profile/account/close_account.dart) which are set up to do the exact same thing.